addons/account_payment/views/account_portal_templates.xml

```diff
@@ -28,6 +28,12 @@
                 <i class="fa fa-check-circle"/> Paid
             </a>
         </xpath>
+        <xpath expr="//div[@id='payment_status']" position="inside">
+            <div t-if="invoice.payment_tx_ids and invoice.amount_total and not success and not error" class="o_account_payment_tx_status" t-att-data-invoice-id="invoice.id">
+                <t t-set="payment_tx_id" t-value="invoice.payment_tx_id"/>
```

Comment:
Couldn't you put the t-set inside the t-call to have it nested inside its own call ? I mean something like
```
<t t-call="payment.bob">
  <t t-set="tx" t-value="alice"/>
</t>
```